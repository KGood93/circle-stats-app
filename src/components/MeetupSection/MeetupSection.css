.MeetupSection {
    margin-bottom: 275px;
}