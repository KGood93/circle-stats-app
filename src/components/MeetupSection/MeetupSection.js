import React, { Component } from 'react'
import MeetupList from '../MeetupList/MeetupList'
import './MeetupSection.css'
//import {Link} from 'react-router-dom'
//import LaunchPage from '../LaunchPage/LaunchPage'

class MeetupSelection extends Component {
    constructor(props) {
        super(props);
        this.state = {
            refresh: false
        };
      }


    render() {
        return (
            <div className="MeetupSection">
                <MeetupList />
                <div className="appDescription">
                    <p className="description">Circle Time Stats is an easy way to keep track of the 
                        attendance for meetup events. Start by selecting the 
                        meetup you would like information on or are hosting an 
                        event for. From there you can either create a new event 
                        or view previous statistics. If you are logging a new 
                        event it is recommended to take a count at the door using 
                        the large counter, with the oops button if you accidentally 
                        click the counter one to many times, also add the location of 
                        the even and any notes you may want on things that can 
                        affect attendance (ie a snow storm). On the stats page you can 
                        view a graph of attendance as well as a table of the stats 
                        including the data, location, number of attendees, and on larger 
                        screens (ie tablet or larger) notes. </p>
                </div>
            </div>
        )
    }
}

export default MeetupSelection

