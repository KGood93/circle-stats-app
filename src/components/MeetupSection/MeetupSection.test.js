import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import MeetupSection from './MeetupSection';

describe('MeetupSection Component', () => {
    it(`renders without crashing`, () => {
        const wrapper = shallow(<MeetupSection />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});