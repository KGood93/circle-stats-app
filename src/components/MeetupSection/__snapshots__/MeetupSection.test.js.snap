// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MeetupSection Component renders without crashing 1`] = `
<div
  className="MeetupSection"
>
  <MeetupList />
</div>
`;
